### DISCUSSION TITLE
Can't use multiple modifier on wire:loading at the same time.

### DISCUSSION THREAD

**Discussion by rahmanramsi** [livewire/livewire#4607] [7. Bugs (archived)] [answered]
I want to show a loading, and use flex modifier for display the loading, at the same time i want to avoid flickering because loading is very fast. but i cant use it at the same time.

already tried `wire:loading.flex.delay' and 'wire:loading.delay.flex' . Nothing working.

**Comment by joshhanley** [comment, marked as answer]
@rahmanramsi they don't work together properly due to limitations of css selectors. 

I'd recommend just adding another div and leaving flex on that div and putting `wire:loading` on the wrapper.

Like

```blade
<div wire:loading.delay>
    <div class="flex">
        ...
    </div>
</div>
```

Hope this helps!
